draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -547,162 +584,173 @@ if rtt_var == NA then rtt_var = rtt_est / 2
+			<t>The flow control algorithm MUST NOT assume the receive capabilities
         of a neighbor are static, i.e., it MUST handle transient conditions
         which result in a slower or faster receive rate on the part of a
         neighbor.</t>
 
-        <t>The flow control algorithm needs to consider the expected delay
-        time in receiving an acknowledgment. See <xref target="LSPACKRate"/>.
+			<t>The flow control algorithm needs to consider the expected delay
+        time in receiving an acknowledgment. See <xref target="LSPACKRate"/>
+.
         This may vary per neighbor.</t>
-		
+
 		</section>
 	</section>
-<section anchor="IANA_Consideration" title="IANA Considerations">
+	<section anchor="IANA_Consideration" title="IANA Considerations">
 
-	<t>IANA is requested to allocate one TLV from the IS-IS TLV codepoint registry.</t>
-	<figure anchor="IANA_Registration" title=''>
-		<preamble></preamble>
-		<artwork align="center">
+		<t>IANA is requested to allocate one TLV from the IS-IS TLV codepoint registry.</t>
+		<figure anchor="IANA_Registration" title=''>
+			<preamble></preamble>
+			<artwork align="center">
    Type    Description                    IIH   LSP   SNP   Purge
    ----    ---------------------------    ---   ---   ---   ---
    TBD1    Flooding Parameters TLV         y     n     y     n
-		</artwork>
-	</figure>
-	<t>This document creates the following sub-TLV Registry:</t>
-	<t>Name: Sub-TLVs for TLV TBD1 (Flooding Parameters TLV).</t>
-	<t>Registration Procedure(s): Expert Review</t>
-	<t>Expert(s): TBD</t>
-	<t>Reference: TBD</t>
-	<texttable anchor="table_ex" title="Initial allocations">
-		<ttcol align='center'>Type</ttcol>
-		<ttcol align='left'>Description</ttcol>
-		<c>0</c>
-		<c>Reserved</c>
-		<c>1</c>
-		<c>LSP Burst Window</c>
-		<c>2</c>
-		<c>LSP Transmission Interval</c>
-		<c>3-255</c>
-		<c>Unassigned</c>
-	</texttable>
-</section>
-<section anchor="Security" title="Security Considerations" toc="default">
-  <t>
-    Security concerns for IS-IS are addressed in <xref target="ISO10589"/>,
-    <xref target="RFC5304"/>, and <xref target="RFC5310"/>.  These documents
+			</artwork>
+		</figure>
+		<t>This document creates the following sub-TLV Registry:</t>
+		<t>Name: Sub-TLVs for TLV TBD1 (Flooding Parameters TLV).</t>
+		<t>Registration Procedure(s): Expert Review</t>
+		<t>Expert(s): TBD</t>
+		<t>Reference: TBD</t>
+		<texttable anchor="table_ex" title="Initial allocations">
+			<ttcol align='center'>Type</ttcol>
+			<ttcol align='left'>Description</ttcol>
+			<c>0</c>
+			<c>Reserved</c>
+			<c>1</c>
+			<c>LSP Burst Window</c>
+			<c>2</c>
+			<c>LSP Transmission Interval</c>
+			<c>3</c>
+			<c>LSPs Per PSNP</c>
+			<c>4</c>
+			<c>Flags</c>
+			<c>5-255</c>
+			<c>Unassigned</c>
+		</texttable>
+	</section>
+	<section anchor="Security" title="Security Considerations" toc="default">
+		<t>
+    Security concerns for IS-IS are addressed in <xref target="ISO10589"/>
+,
```

Comment:
Another bad line break resulting in bad formatting.

Reply:
ack